docs/section5/Rule5-47.md

```diff
@@ -0,0 +1,43 @@
+
+# Envelope - Rule 5-47  
+
+**Rule ID:** 5-47  
+**Rule Description:** The baseline air leakage rate of the building envelope (I75Pa) at a fixed building pressure differential of 0.3 in. of water shall be 1 cfm/ft2.  The air leakage rate of the building envelope shall be converted to appropriate units for the simulation program using one of the methods in Section G3.1.1.4.  
+**Rule Assertion:** Sum of B-RMR infiltration.infiltration.air_leakage_rate = expected value.  
+**Appendix G Section:** Section 5 Envelope  
+**Appendix G Section Reference:** Section G3.1-5(h) Building Envelope Modeling Requirements for the Baseline building  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:** 
+
+  1. get_surface_conditioning_category()
+  2. get_zone_conditioning_category()
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dict_b = get_surface_conditioning_category(B_RMR)`
+
+- Get zone conditioning category dictionary for B_RMR: `zone_conditioning_category_dict_b = get_zone_conditioning_category(B_RMR)`
+
+- For each zone in the Baseline model: `for zone_b in B_RMR...zones:`
+
+  - For each surface in zone: `for surface_b in zone_b.surfaces:`
+
+    - Check if surface is regulated, add zone total area of building envelope to building total: `if scc_dict_b[surface_b.id] != "UNREGULATED": building_total_envelope_area += sum(surface.area for surface in zone_b.surfaces)`
+
+  - Check if zone is conditioned or semi-heated, add zone infiltration flow rate to building total: `if zone_conditioning_category_dict_b[zone.id] in [CONDITIONED RESIDENTIAL, CONDITIONED NON-RESIDENTIAL, CONDITIONED MIXED, SEMI-HEATED]: building_total_air_leakage_rate_b += zone_b.infiltration.infiltration_flow_rate`
+
+- Calculate the required baseline infiltration flow rate in cfm: `target_infiltration_flow_rate_b = 1.0 * building_total_envelope_area`
+
+**Rule Assertion:**  
+
+- Case 1: For B_RMR, if the total zone infiltration rate for conditioned and semi-heated zones is equal to the required baseline infiltration rate at wind pressure: `if building_total_air_leakage_rate_b == target_infiltration_flow_rate_b: PASS`
```

Comment:
The target flow rate calculated in line #35 is CFM @ 75PA. However, building_total_air_leakage_rate_b calculated in #34 is the sum of  zone.infiltration.infiltration_flow_rate, which is the rate at wind pressure. So I think the assertion should be 
if building_total_air_leakage_rate_b == target_infiltration_flow_rate_b / 0.112 : PASS

Reply:
Corrected.